app/javascript/react/App.tsx

```diff
@@ -1,14 +1,18 @@
 import React from "react";
-import './locales/i18n'
+import { Provider } from "react-redux";
 
 import { CalendarPage } from "./pages/CalendarPage";
 import GlobalStyles from "./assets/styles/global-styles";
+import store from "./store/index";
+import './locales/i18n'
 
 const App = () => {
   return (
     <>
-      <GlobalStyles />
-      <CalendarPage />
+      <Provider store={store}>
```

Comment:
An empty tag is not needed anymore, the rule of returning a single child element is et thanks to adding the Provider component.